Conceptpower+Spring/src/test/java/edu/asu/conceptpower/servlet/rest/ConceptLookUpTest.java

```diff
@@ -0,0 +1,149 @@
+import edu.asu.conceptpower.app.util.URIHelper;
+import edu.asu.conceptpower.core.ConceptEntry;
+import edu.asu.conceptpower.core.ConceptType;
+import edu.asu.conceptpower.rest.ConceptLookup;
+import edu.asu.conceptpower.rest.msg.IMessageConverter;
+import edu.asu.conceptpower.rest.msg.IMessageRegistry;
+import edu.asu.conceptpower.rest.msg.json.JsonConceptMessage;
+import edu.asu.conceptpower.rest.msg.xml.XMLConceptMessage;
+import junit.framework.Assert;
+
+public class ConceptLookUpTest {
+
+    @Mock
+    private IConceptManager dictManager;
+
+    @Mock
+    private TypeDatabaseClient typeManager;
+
+    @Mock
+    private IMessageRegistry messageFactory;
+
+    @InjectMocks
+    private ConceptLookup conceptLookup;
+
+    @Mock
+    private RDFMessageFactory rdfFactory;
+
+    @Mock
+    private IMessageConverter xmlMEssageFactory;
+
+    @Mock(name = "jsonMessageFactory")
+    private IMessageConverter jsonMessageFactory;
+
+    @Mock
+    private URIHelper uriCreator;
+
+    @Before
+    public void setup() throws IOException, IllegalAccessException, LuceneException, IndexerRunningException {
+        MockitoAnnotations.initMocks(this);
+        String word = "pony";
+        String pos = POS.NOUN;
+        String typeId1 = "TYPE-1";
+        String typeId2 = "TYPE-2";
+
+        ConceptEntry[] entries = new ConceptEntry[2];
+        ConceptEntry entry = new ConceptEntry();
+        entry.setWord(word);
+        entry.setPos(pos);
+        entry.setId("WID-123");
+        entry.setConceptList(Constants.WORDNET_DICTIONARY);
+        entry.setTypeId("TYPE-1");
+        entries[0] = entry;
+
+        ConceptEntry entry2 = new ConceptEntry();
+        entry2.setWord(word);
+        entry2.setPos(pos);
+        entry2.setConceptList("Test List");
+        entry2.setTypeId("TYPE-2");
+        entries[1] = entry2;
+
+        ConceptType type1 = new ConceptType();
+        type1.setTypeId(typeId1);
+        type1.setDescription("TypeID 1");
+        type1.setTypeName("TYPE-1");
+
+        ConceptType type2 = new ConceptType();
+        type2.setTypeId(typeId2);
+        type1.setDescription("TypeID 2");
+        type1.setTypeName("TYPE-2");
+
+        Mockito.when(dictManager.getConceptListEntriesForWordPOS("pony", POS.NOUN, null)).thenReturn(entries);
+        Mockito.when(typeManager.getType(typeId1)).thenReturn(type1);
+        Mockito.when(typeManager.getType(typeId2)).thenReturn(type2);
+        Mockito.when(messageFactory.getMessageFactory(MediaType.APPLICATION_XML_VALUE)).thenReturn(xmlMEssageFactory);
+        Mockito.when(messageFactory.getMessageFactory(MediaType.APPLICATION_JSON_VALUE)).thenReturn(jsonMessageFactory);
+
+        Mockito.when(xmlMEssageFactory.createConceptMessage()).thenReturn(new XMLConceptMessage(uriCreator));
+        Mockito.when(jsonMessageFactory.createConceptMessage()).thenReturn(new JsonConceptMessage(uriCreator));
+    }
+
+    @Test
+    public void getWordNetEntryTest()
```

Comment:
please follow our naming conventions for tests:
https://diging.atlassian.net/wiki/display/DIGING/Unit+Tests